text_processing.py

```diff
@@ -171,8 +171,10 @@ def move(self, copy_instead):
         arg2 = self.speech[to_index+4:]
         dest = self.__common_dest(arg2)
         if dest == RETURN_ERROR:
-            print("Couldn't recognize destination. Trying as absolute path")
-            dest = arg2
+            # print("Couldn't recognize from common destination. Trying as absolute path")
+            # trying the present working directory
+            curr_dir = os.getcwd()
+            dest = curr_dir + "/" + arg2
```

Comment:
we want to eventually allow for absolute paths. Currently this works for relative path from pwd anyway, do we need this change?